### DISCUSSION TITLE
Screenshot from external screen is dimmer when MonitorControl is set to `Avoid gamma table manipulation`?

### DISCUSSION THREAD

**Discussion by Paul-B** [MonitorControl/MonitorControl#866] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

When I take a screenshot of anything that is displayed on the external screen then it looks dimmer. As if it has a transparent black overlay. 

### Steps to reproduce

-Take screenshot
-Open in preview (or anywhere else)

### Expected behavior

Normal brightness screenshot

### Anything else?

From external screen:
![image](https://user-images.githubusercontent.com/1169385/147583165-2ae0c8d8-d89b-45f3-b9ec-7732c501e9c9.png)

After moving to laptop screen
<img width="348" alt="image" src="https://user-images.githubusercontent.com/1169385/147583219-666d851d-53c0-415e-af0a-b7b541e0277d.png">

### Environment Information (please complete the following information)

```markdown
- macOS version:12.1
- Mac model:Air
- MonitorControl version:4.0.2
- Monitor(s):Lenovo Q27h-10
- Apple Silicon/M1 (yes or no): yes
- USB-C
```

**Comment by waydabber** [comment, marked as answer]
This is normal if you use "Avoid gamma table manipulation" under Displays. You need to enable this option only if you are using flux and the two apps are in conflict or you are using DisplayLink which does not support gamma table manipulation.

**Reply by Paul-B** [reply]
Thank you this solves it. 
I do not remember enabling this setting, I use it as much as default as possible. Did I miss something obvious in the docs? 

I do see this:
Note to f.lux users - please activate Avoid gamma table manipulation under Preferences » Displays! This step is not needed if you use Night Shift.

**Reply by waydabber** [reply]
This is not be the default setting, maybe you activated it by accident? The app activates this automatically if it detects conflict with an other app (like flux) but when this happens there is also a pop-up explaining what is wrong.

**Reply by Paul-B** [reply]
I must have done it by accident. Indeed going for default by pressing reset also makes it go away. I do not use f.lux so Im in the dark why it was on.
Anyway, thank you for your help and maintaining this app, it is very useful! The OSD menu of the Lenovo screen is terrible and now I can just click a setting!
